spec.html

```diff
@@ -4482,6 +4483,21 @@ <h1>Strict Equality Comparison</h1>
   <emu-clause id="sec-operations-on-objects">
     <h1>Operations on Objects</h1>
 
+    <emu-clause id="sec-allocatebasicobject" aoid="AllocateBasicObject">
+      <h1>AllocateBasicObject ( _internalSlotsList_ [ , _proto_ ])</h1>
+      <p>The abstract operation AllocateBasicObject is the source of all ECMAScript objects, including both ordinary objects and exotic objects. It factors out common steps used in creating all objects, and centralizes object creation.</p>
+
+      <emu-alg>
+        1. Let _obj_ be a newly created object with an internal slot for each name in _internalSlotsList_.
+        1. Set _obj_'s essential internal methods to the default ordinary object definitions specified in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>.
```

Comment:
I think this should explicitly say that after returning from this operation only the direct caller of
AllocateBaseObject may modify the definition of the returned object's essential internal methods. 

Basely, we don't what to imply that it is possible for internal method definitions to change at arbitrary times during the entire lifetime of an object.

Reply:
Done, thank you! Also added more asserts on callers relating to the interaction with non-overridden methods.